configs/14.0/packages/glibc/sources

```diff
@@ -1,53 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
It caught my attention to see 14 pointing to a future version (next). But it's explained in the commit message. Makes sense. Thanks!